bridges/zap/zap_logger.go

```diff
@@ -0,0 +1,119 @@
+package zap
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/log/noop"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	bridgeName = "go.opentelemetry.io/contrib/bridge/zapcore"
+)
+
+type OtelZapCore struct {
```

Comment:
Similar here regarding naming.

```suggestion
type Core struct {
```